lucida/commandcenter/controllers/Config.py

```diff
@@ -12,41 +12,163 @@
 # models are already saved in `../models`.
 TRAIN_OR_LOAD = 'train' # either 'train' or 'load'
 
+
```

Comment:
if this is a demo workflow, should be named as such and maybe in its own directory?